adam/experiment/observer.py

```diff
@@ -326,12 +330,31 @@ def post_observer_log(
                     f'\t\t\t\t<td valign="top">{learner_description}<br/>Accuracy: {accuracy:2.2f}</td>\n'
                 )
 
+            if situation and isinstance(situation, HighLevelSemanticsSituation):
+                render_buttons_text = self.render_buttons_html(situation)
+            else:
+                render_buttons_text = ""
+
             outfile.write(
                 f'\t\t\t\t<td valign="top">{clickable_perception_string}\n\t\t\t\t</td>\n'
+                f"\t\t\t\t<td valign='top'>{render_buttons_text}</td>"
                 f"\t\t\t</tr>\n\t\t</tbody>\n\t</table>"
             )
             outfile.write("\n</body>")
 
+    def render_buttons_html(self, situation: HighLevelSemanticsSituation) -> str:
+        # for now every item is going to have 3 buttons for three frames and later we might corral that
+        buttons = [
```

Comment:
Can we make this just one frame for static situations? Or is that hard?

Reply:
Now that you mention it, there does appear to be a `is_dynamic` field in the HighLevelSemanticsSituation` class